packages/create-vite/template-vanilla-ts/src/counter.ts

```diff
@@ -0,0 +1,9 @@
+export function setupCounter(element: HTMLDivElement) {
```

Comment:
```suggestion
export function setupCounter(element: HTMLButtonElement) {
```
It seems it was applied once but it was reverted for some reason?